src/Hosting/Server.IntegrationTesting/src/Common/DeploymentResult.cs

```diff
@@ -64,6 +64,6 @@ public DeploymentResult(ILoggerFactory loggerFactory, DeploymentParameters deplo
         /// <param name="baseHandler"></param>
         /// <returns></returns>
         public HttpClient CreateHttpClient(HttpMessageHandler baseHandler) =>
-            new HttpClient(new LoggingHandler(_loggerFactory, baseHandler)) { BaseAddress = new Uri(ApplicationBaseUri) };
+            new(new LoggingHandler(_loggerFactory, baseHandler)) { BaseAddress = new Uri(ApplicationBaseUri) };
```

Comment:
This is a coding style change we have not adopted yet. We don't usually accept changes that are only style based. Please revert.

Reply:
OK. Reverted.